### PR TITLE
Add tests for tensor data and add random-bigint package

### PR DESCRIPTION
**Pull request by almilosz**
### Details:
 - Add tests for tensor data and getData()
- Add *random-bigint* package as devDependency

### Tickets:
 - 117419

### CODE REVIEW

File: src/bindings/js/node/tests/tensor.test.js [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+
+let ov = require('../build/Release/ov_node_addon.node');
 const assert = require('assert');
 const { test, describe, it } = require('node:test');
+const random = require('random-bigint');
 
 const shape = [1, 3, 224, 224];
 const elemNum = 1 * 3 * 224 * 224;
 const data = Float32Array.from({length: elemNum}, () => Math.random() );
+const ets = [
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/tests/tensor.test.js:13]: How can I decode `ets`?

File: src/bindings/js/node/tests/tensor.test.js [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+// SPDX-License-Identifier: Apache-2.0
+
+let ov = require('../build/Release/ov_node_addon.node');
 const assert = require('assert');
 const { test, describe, it } = require('node:test');
+const random = require('random-bigint');
 
 const shape = [1, 3, 224, 224];
 const elemNum = 1 * 3 * 224 * 224;
 const data = Float32Array.from({length: elemNum}, () => Math.random() );
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/tests/tensor.test.js:12]:
```suggestion
const data = Float32Array.from({ length: elemNum }, () => Math.random() );
```

File: src/bindings/js/node/tests/tensor.test.js [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
+const ets = [
+  [ov.element.i8, 'i8', Int8Array.from({length: elemNum}, () => Math.random() )],
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/tests/tensor.test.js:14]: Here and below:
```suggestion
  [ov.element.i8, 'i8', Int8Array.from({ length: elemNum }, () => Math.random() )],
```

File: src/bindings/js/node/tests/tensor.test.js [hunk 4]
```diff
@@ -1,10 +1,27 @@
-var ov = require('../build/Release/ov_node_addon.node');
+// -*- coding: utf-8 -*-
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+let ov = require('../build/Release/ov_node_addon.node');
 const assert = require('assert');
 const { test, describe, it } = require('node:test');
+const random = require('random-bigint');
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/tests/tensor.test.js:8]: Something meaningful 
```suggestion
const getRandomBigInt = require('random-bigint');
```

File: src/bindings/js/node/tests/tensor.test.js [hunk 5]
```diff
@@ -1,10 +1,27 @@
-var ov = require('../build/Release/ov_node_addon.node');
+// -*- coding: utf-8 -*-
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+let ov = require('../build/Release/ov_node_addon.node');
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/tests/tensor.test.js:5]: Maybe I ask this question all the time., sorry in advance) Why is let, not const?

**Review by vishniakov-nikolai — APPROVED** [review]

File: src/bindings/js/node/tests/tensor.test.js (hunk 1 above)
Comment (almilosz) [reply src/bindings/js/node/tests/tensor.test.js:13]: Those are parameters for tests, so I changed the name to `params`

File: src/bindings/js/node/tests/tensor.test.js (hunk 5 above)
Comment (almilosz) [reply src/bindings/js/node/tests/tensor.test.js:5]: Sorry 😄 Those parts of code are old (~1 year) and I am mostly focused on new changes

**Pull request merged**